homeassistant/components/rhasspy/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""The Rhasspy integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+__all__ = ["DOMAIN", "async_setup_entry", "async_unload_entry"]
```

Comment:
I think Paulus does this to export the DOMAIN so it can be imported from here?
A bit odd, as it's not a public API outside of this package itself.

Reply:
`DOMAIN` was a required field to be set because we used to not have manifests. But uh, now that I am looking at it, I can't find that check anymore 🙃